cli/cmd/event.go

```diff
@@ -152,6 +188,18 @@ func init() {
 	eventListCmd.Flags().StringVar(&eventsCmdState.End,
 		"end", "", "end of the time range in UTC (format: yyyy-MM-ddTHH:mm:ssZ)",
 	)
+	// add days flag to events list command
+	eventListCmd.Flags().IntVar(&eventsCmdState.Days,
+		"days", 0, "list events from a specific number of days (max: 7 days)",
+	)
+	// add severity flag to events list command
+	eventListCmd.Flags().StringVar(&eventsCmdState.Severity,
+		"severity", "",
+		fmt.Sprintf(
+			"list events with a specific severity (%s)",
```

Comment:
@scottford-lw could you please verify that these messages are clear? Any recommendations will be appreciated!

The current help output looks like this:
```
$ lacework events list --help
List all events from a time range, by default this command displays the
events from the last 7 days, but it is possible to specify a different
time range.

Usage:
  lacework event list [flags]

Flags:
      --days int          list events from a specific number of days (max: 7 days)
      --end string        end of the time range in UTC (format: yyyy-MM-ddTHH:mm:ssZ)
  -h, --help              help for list
      --severity string   list events with a specific severity (critical, high, medium, low, info)
      --start string      start of the time range in UTC (format: yyyy-MM-ddTHH:mm:ssZ)
```

Reply:
@afiune The help messages look great to me! 

![](https://media3.giphy.com/media/5wWf7GW1AzV6pF3MaVW/giphy.gif)